features/step_definitions/record_of_guidance_steps.rb

```diff
@@ -2,7 +2,11 @@
   page = AppointmentSummaryPage.new
   page.load
   page.name.set 'Joe Bloggs'
+  page.email_address.set 'joe.bloggs@example.com'
   page.submit.click
 
-  expect(AppointmentSummary.count).to eql(1)
+  appointment_summary = AppointmentSummary.last
```

Comment:
There is a race condition here (test runner vs application), but its probably fine for now.